### DISCUSSION TITLE
初始化从git拉取文件总是失败，搭梯子也没用

### DISCUSSION THREAD

**Discussion by iu-x** [vuepress-reco/vuepress-theme-reco#37] [Q&A]
![image](https://user-images.githubusercontent.com/80145954/157584666-a5509bd9-a12b-4772-b1ce-7dd18a6c1535.png)

**Comment by recoluan** [comment]
这个真的没有好办法呢，你也可以手动下载 [2.x demo](https://github.com/vuepress-reco/vuepress-theme-reco-demo/tree/demo/2.x)。
